requirements-app.txt

```diff
@@ -9,4 +9,4 @@ lxml==4.2.5
 
 git+https://github.com/alphagov/digitalmarketplace-content-loader.git@5.0.0#egg=digitalmarketplace-content-loader==5.0.0
 git+https://github.com/alphagov/digitalmarketplace-utils.git@45.0.2#egg=digitalmarketplace-utils==45.0.2
-git+https://github.com/alphagov/digitalmarketplace-apiclient.git@19.7.0#egg=digitalmarketplace-apiclient==19.7.0
+git+https://github.com/alphagov/digitalmarketplace-apiclient.git@19.10.0#egg=digitalmarketplace-apiclient==19.10.0
```

Comment:
Might be worth putting something in the commit message saying why the api client bump is needed

Reply:
Now done 👍